config/config.py.template

```diff
@@ -500,9 +500,15 @@ MLLIB_DECISION_TREE_TEST_OPTS = MLLIB_COMMON_OPTS + [
 ]
 
 if MLLIB_SPARK_VERSION >= 1.2:
+    ensembleTypes = ["mllib.RandomForest"]
+    if MLLIB_SPARK_VERSION >= 1.3:
+        ensembleTypes.append("mllib.GradientBoostedTrees")
```

Comment:
This was in 1.2, so it should be OK to leave in 1.2.

Let's use the "ml." prefix for spark.ml tests, but not add a prefix to spark.mllib tests.  That way, we won't break the old versions.